pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <!-- We use the PackageVersion property defined in Packaging.props for the Prerelease packages which are using it to be built --> 
     <_PreReleasePackageVersion>$(PackageVersion)</_PreReleasePackageVersion> 
-    <PackageVersion>3.1.0</PackageVersion>
-    <ServiceModelVersion>4.8.1</ServiceModelVersion>
+    <PackageVersion>3.1.1</PackageVersion>
+    <ServiceModelVersion>4.7.0</ServiceModelVersion>
```

Comment:
This is downgrading ServiceModelVersion, was that intentional?  I believe that's a breaking change.

Reply:
yes we never shipped any serviceModelVersion package with 4.8.1 or 4.8.0.
The latest package ic ould find in nuget was 4.7.0

Reply:
https://github.com/dotnet/corefx/pull/42952#discussion_r454806663

Reply:
I see, you were saying we updated it but never shipped that.  Wouldn't we want to ship that now or does the new version have a problem?

Reply:
We are shipping the Microsoft.Windows.Compatibility package and its version has been incremented.
We are not shipping the serviceModelVersion packages so we are just correcting the version to the last package persent. ServiceModel packages are used as package references in compatibility package

Reply:
Got it.